jaeger-jaxrs2/src/main/java/com/uber/jaeger/filters/jaxrs2/ClientFilter.java

```diff
@@ -79,6 +84,7 @@ public void filter(ClientRequestContext clientRequestContext, ClientResponseCont
             }
         } catch (Exception e) {
             logger.error("Client Filter Response:", e);
+            metrics.clientFilterExceptionResponse.inc(1);
```

Comment:
I don't understand where this exception can come from. `getProperty`, `getStatus` ?